### DISCUSSION TITLE
Keyboard Shortcut Visibility

### DISCUSSION THREAD

**Discussion by jensenautomation** [microsoft/terminal#17626] [Ideas]
I have added a number of profiles and have explicitly deleted their associated Action which by default is ctrl+shift+n. However, these keyboard shortcuts still appear in the profile dropdown list. I would like to propose adding a setting in the Settings, Appearance section to either globally turn off the visibility of keyboard shortcuts or allow the user to turn off specific categories, like profiles.

**Comment by zadjii-msft** [comment]
There's a bug laying around somewhere on the backlog for that. What version of the Terminal are you on? I believe that might have been fixed either in 1.21 or early in 1.22

**Reply by zadjii-msft** [reply]
https://github.com/microsoft/terminal/issues/13943 was the one I was thinking of

**Comment by jensenautomation** [comment]
I am on v1.20.11781.0. Looks like the keybinding issue is addressed in #13943 but I can't install a pre-release version unfortunately due to company policies. Are there plans to add some setting to change visibility of these keyboard shortcuts in the near future?

**Reply by DHowett** [reply]
There is no plan for that at this moment, nope!

**Reply by zadjii-msft** [reply]
To clarify: the issue you described is fixed in canary builds, but not stable. We don't have a plan to backport those changes in Preview back to the stable channel before _we just promote them to stable_. 

I'd guess it's probably ~6mos before 1.22 becomes stable (which would be the first stable release with #17215)

**Reply by jensenautomation** [reply]
Thanks for the clarification. 

Is there a place I can add this as a feature request?

**Reply by DHowett** [reply]
This is the right place! We're liable to put it on the backlog, though - it's not something that many (any?) folks have asked for.

**Reply by jensenautomation** [reply]
Ok, thank you!
